hedyorg/hedy
# [UI] added grid_overview button in the class page "for-teachers/class/class_id" that forwards into /class/grid_overview/cl…

**Description**
A new button under teacher's view "for-teachers/class/class_id" that forwards into a new page "grid_overview/class/class_id" that will hold the grid. Another button in the new page that allows you to go back to the class page "for-teachers/class/5c39c2a936f24db1a4935c52fab77cd7"

**Fixes _issue or discussion number_**
First step in regards to discussion number 4008

**How to test**
Log into teacher's view -> for_teachers -> click on one of the classes -> grid_overview button. You should end up at http://localhost:8080/grid_overview/class/class_id. You can also test the other button by and you should end up back in the class page
